tests/drivers/uart/uart_async_api/boards/nucleo_h743zi.conf

```diff
@@ -1 +1,2 @@
 CONFIG_NOCACHE_MEMORY=y
+CONFIG_DCACHE=n
```

Comment:
is `CONFIG_NOCACHE_MEMORY=y` still required for this testcase ?

Reply:
Please see the above comment.